### PR TITLE
Fine tune Monitoring guide (big)

### PR DESCRIPTION
**Pull request by Lennonka**
Polishing the new Monitoring guide.
**Recommendation for reviewers**: Check out the preview for smooth reading experience.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9 (Satellite 6.15) if available
* That's all.

### CODE REVIEW

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-For more details on using Grafana, see the https://grafana.com/[Grafana Labs] website.
+You can also select the time span displayed for each widget.
 
 .Procedure
-. Open the following URL:
-http://_{foreman-example-com}_:3000
+* Open the following URL in a browser: http://_{foreman-example-com}_:3000
 
-.Example of the Grafana dashboard
-image::grafana_screenshot.png[PCP Redis Host Overview dashboard]
```
Comment (Lennonka) [inline guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:16]: We generally try to avoid screenshots. Can I have an ack from peers?

File: guides/common/attributes-titles.adoc [hunk 2]
```diff
@@ -18,7 +18,7 @@
 :ManagingHostsDocTitle: Managing Hosts
 :ManagingOrganizationsLocationsDocTitle: Managing Organizations and Locations in {ProjectName}
 :ManagingSecurityDocTitle: Managing Security Compliance
-:MonitoringDocTitle: Monitoring {ProjectName}
+:MonitoringDocTitle: Monitoring {Project} Performance
```
Comment (evgeni) [inline guides/common/attributes-titles.adoc:21]: This should also be updated in `web/content/js/versions.js`

**Review by evgeni — CHANGES_REQUESTED** [review]
I could not spot any technical changes, so ACK on that side :)
"Request changes" just because the title was not updated in `versions.js`

File: guides/common/modules/proc_retrieving-archived-metrics-using-cli.adoc [hunk 3]
```diff
@@ -26,8 +24,7 @@ To do that, add the `--archive` parameter and specify the archive file.
 # pmval --archive /var/log/pcp/pmlogger/_{foreman-example-com}_/_20230831.00.10_ \
   -f 1 disk.partitions.write
 ----
-
-* To list disk write operations per partition, with a two second interval, between the time period 14:00 and 14:15:
+* To list disk write operations per partition, with 2-second interval, over the time period between 14:00 and 14:15:
```
Comment (adamlazik1) [inline guides/common/modules/proc_retrieving-archived-metrics-using-cli.adoc:27]:
```suggestion
* To list disk write operations per partition, with a 2-second interval, over the time period between 14:00 and 14:15:
```

File: guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc [hunk 4]
```diff
@@ -1,19 +1,24 @@
 [id='retrieving-live-metrics-using-cli_{context}']
-= Retrieving Live Metrics using CLI
+= Retrieving Live Metrics in the CLI
```
Comment (adamlazik1) [inline guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc:2]: Imho "using" was better here, but should be "using the"

File: guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc [hunk 5]
```diff
@@ -1,19 +1,24 @@
...
+PCP converts the number of writes to disk partitions from a counter value to a rate value.
+The `-f 1` argument specifies to abbreviate the values to one decimal place.
++
+Example output:
++
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
 metric:    disk.partitions.write
-host:      {foreman-example-com}
+host:      _{foreman-example-com}_
```
Comment (adamlazik1) [inline guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc:21]: Since this is example output, it can stay non-italicized, but I don't care either way.

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc [hunk 6]
```diff
@@ -1,16 +1,13 @@
 [id='retrieving-metrics-in-the-web-ui_{context}']
 = Retrieving Metrics in the Web UI
 
-The Grafana web application provides a dashboard-style view, with default widgets displaying the values of metrics.
+You can view the Grafana dashboard, which provides default widgets displaying the values of metrics.
```
Comment (adamlazik1) [inline guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]:
```suggestion
You can view the Grafana dashboard which provides default widgets displaying the values of metrics.
```

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc (hunk 1 above)
Comment (adamlazik1) [reply guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:16]: Yeah.

File: guides/common/modules/proc_verifying-pcp-configuration.adoc [hunk 7]
```diff
@@ -8,23 +8,23 @@ To verify PCP is configured correctly, and services are active, run the followin
 ----
 
 This outputs a summary of the active PCP configuration.
+Example output from the `pcp` command:
```
Comment (adamlazik1) [inline guides/common/modules/proc_verifying-pcp-configuration.adoc:11]:
```suggestion
Example output of the `pcp` command:
```

**Review by adamlazik1 — COMMENTED** [review]
Looks good overall. Pointed out some minor things.

File: guides/common/attributes-titles.adoc (hunk 2 above)
Comment (Lennonka) [reply guides/common/attributes-titles.adoc:21]: Thank you!

File: guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc (hunk 4 above)
Comment (Lennonka) [reply guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc:2]: It actually should be "by using the CLI" for complete clarity, which is so damn long :D I was thinking about leaving it out altogether. What do you think about that?

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc (hunk 6 above)
Comment (Lennonka) [reply guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]: A comma before "which" should be correct, because it introduces a nonrestrictive clause.
https://www.merriam-webster.com/grammar/when-to-use-that-and-which

File: guides/common/modules/proc_verifying-pcp-configuration.adoc [hunk 8]
```diff
@@ -25,6 +25,6 @@ Performance Co-Pilot configuration on {foreman-example-com}:
      pmie: primary engine: /var/log/pcp/pmie/{foreman-example-com}/pmie.log
 ----
 
-In this example, both the Performance Metrics Collector Daemon (pmcd), and the Performance Metrics Web Daemon (pmwebd) services are running.
-It also confirms the PMDAs which are collecting metrics.
-Finally, it lists the currently actively archive file, in which `pmlogger` is storing metrics.
+In this example, both the Performance Metrics Collector Daemon (pmcd) and Performance Metrics Web Daemon (pmwebd) services are running.
```
Comment (Lennonka) [inline guides/common/modules/proc_verifying-pcp-configuration.adoc:28]: @evgeni FYI, this is a bit confusing, because `pmwebd` is not in the example output above. Are we missing a piece of the output? If not, then do we have to mention this daemon, when it isn't in the example?

Comment (evgeni) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:28]: Huh, this one did slip in. Yeah, pmwebd is not a thing anymore in PCP5+, let me see if we need to mention proxy or nothing.

Comment (evgeni) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:28]: Yeah, it should be the proxy. So I guess we should write "Proxy for Performance Metrics Collector Daemon (pmproxy)" or "Performance Metrics Proxy Daemon (pmproxy)" or something similar (that wording is taken from their own docs: https://man7.org/linux/man-pages/man1/pmproxy.1.html https://pcp.readthedocs.io/en/latest/QG/QuickReferenceGuide.html#pcp-web-services )

File: guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc (hunk 4 above)
Comment (adamlazik1) [reply guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc:2]: Yeah that works.

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc (hunk 6 above)
Comment (adamlazik1) [reply guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]: I see :+1:

File: guides/common/modules/proc_verifying-pcp-configuration.adoc [hunk 9]
```diff
@@ -1,15 +1,15 @@
...
 
-To verify PCP is configured correctly, and services are active, run the following command:
+To verify PCP is configured correctly and services are active, run the following command:
 
 ----
 # pcp
 ----
 
 This outputs a summary of the active PCP configuration.
+Example output of the `pcp` command:
```
Comment (adamlazik1) [inline guides/common/modules/proc_verifying-pcp-configuration.adoc:11]:
```suggestion

Example output of the `pcp` command:
```

**Review by adamlazik1 — APPROVED** [review]
LGTM stylewise. One last nitpick.

File: guides/common/modules/proc_verifying-pcp-configuration.adoc (hunk 8 above)
Comment (Lennonka) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:28]: Perfect, thanks!

File: guides/common/modules/proc_verifying-pcp-configuration.adoc (hunk 9 above)
Comment (Lennonka) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:13]: @adamlazik1 I did it a bit differently, let me know if it works.

File: guides/common/modules/con_metrics-overview.adoc [hunk 10]
```diff
@@ -1,19 +1,18 @@
 [id='metrics-overview_{context}']
 = Metrics Overview
 
-Obtaining metrics from {Project} is useful for troubleshooting a current issue, and capacity planning.
-This guide describes how to collect live metrics and archive them for a fixed period of time.
+You can set up a solution to collect live metrics from {ProjectServer}, archive them for a fixed period of time, and analyze them.
```
Comment (maximiliankolb) [inline guides/common/modules/con_metrics-overview.adoc:4]: What does "solution" mean? Is there a more descriptive word? Or maybe "... set up a third party solution to ..."? or "external solution"?

File: guides/common/modules/proc_changing-default-logging-interval.adoc [hunk 11]
```diff
@@ -1,9 +1,17 @@
 [id='changing-default-logging-interval_{context}']
 = Changing Default Logging Interval
 
-This procedure describes how to change the default logging interval.
+You can change the default logging interval to either increase or decrease the sampling rate, at which the PCP metrics are logged.
+A larger interval means a lower sampling rate.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_changing-default-logging-interval.adoc:5]: means > results in?

File: guides/common/modules/proc_retrieving-archived-metrics.adoc [hunk 12]
```diff
@@ -1,17 +1,16 @@
-[id='retrieving-archived-metrics-using-cli_{context}']
-= Retrieving Archived Metrics using CLI
+[id='retrieving-archived-metrics_{context}']
+= Retrieving Archived Metrics
 
 You can use the PCP CLI tools to retrieve metrics from an archive file.
-To do that, add the `--archive` parameter and specify the archive file.
 
-* To list all metrics which were enabled when the archive file was created, enter the following command:
+.Procedures
```
Comment (maximiliankolb) [inline guides/common/modules/proc_retrieving-archived-metrics.adoc:6]: Maybe reword to "Examples"? I like that you went with an unordered list here.

File: guides/common/modules/proc_viewing-data-storage-statistics.adoc [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+You can list all available metrics, grouped by the frequency at which they are logged.
+For each group, you can also view the storage required to store the listed metrics, per day.
+
+Example storage statistics:
+----
+logged every 60 sec: 61752 bytes or 84.80 Mbytes/day
+----
+
+.Procedure
+* To view data storage statistics, enter the following command:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_viewing-data-storage-statistics.adoc:13]: "... on your {ProjectServer}"?

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc (hunk 6 above)
Comment (maximiliankolb) [inline guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]: this sounds off to me. Maybe "... view the Grafana dashboard to view widgets ..."?

**Review by maximiliankolb — COMMENTED** [review]
Thanks Lena! Left some suggestions; none of them are blocking this PR. I cannot comment too much on the actual content though.

File: guides/common/modules/proc_verifying-pcp-configuration.adoc (hunk 9 above)
Comment (adamlazik1) [reply guides/common/modules/proc_verifying-pcp-configuration.adoc:13]: That works, thanks!

File: guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc (hunk 6 above)
Comment (Lennonka) [reply guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc:4]: How about: "... view the Grafana dashboard, which displays the values of metrics as widgets." Would that work better?
